src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -92,9 +94,13 @@ export class TransactionSummary extends React.Component<
         <StackableBorderBox flexDirection="column">
           {isOfferFlow ? (
             <>
-              <Entry label="Your offer" value={offerTotal} />
+              <Entry label="Your offer" value={offerOverride} />
```

Comment:
I think this should be `<Entry label="Your offer" value={offerOverride || offerTotal} />`